internal/provisioner/cluster.go

```diff
@@ -30,15 +30,20 @@ func CreateCluster(provider, size string, logger log.FieldLogger) error {
 		return fmt.Errorf("unsupported size %s", size)
 	}
 
-	outputDir := "tmp"
-	_, err := os.Stat("tmp")
+	if clusterId == "" {
+		clusterId = model.NewId()
+	}
+
+	// Temporarily relocate the kops output directory to a local folder based on the
+	// cluster name. This won't be necessary once we persist the output to S3 instead.
+	outputDir := fmt.Sprintf("cluster-%s", clusterId)
```

Comment:
Should we locate this under `clusters/cluster-%s` just to tidy things up for the local developer, and then update `.gitignore` to remove `tmp` and add `clusters`?